main.tf

```diff
@@ -12,8 +12,8 @@ resource "aws_route53_record" "this" {
   name            = aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_name"]
   type            = aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_type"]
   zone_id         = element(var.zone_ids, count.index)
-  records = aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_value"]
-  ttl     = 60
+  records         = [aws_acm_certificate.this.domain_validation_options[count.index]["resource_record_value"]]
```

Comment:
Salut @feraudet Merci pour votre contribution :)
According to the updated docs https://www.terraform.io/docs/providers/aws/r/route53_record.html the records declaration is ok with brackets. I am merging this to the master, and we'll create a new tag.
à bientôt!

Reply:
Merci :)